plugins/hashsum_download/server/__init__.py

```diff
@@ -17,19 +17,24 @@
 #  limitations under the License.
 ###############################################################################
 
+import hashlib
+import six
+
+from girder import events
 from girder.api import access
 from girder.api.describe import autoDescribeRoute, Description
 from girder.api.rest import RestException, setRawResponse, setResponseHeader
 from girder.api.v1.file import File
 from girder.constants import AccessType, TokenScope
 from girder.utility.model_importer import ModelImporter
 
+SUPPORTED_ALGORITHMS = {'sha256'}
+
 
 class HashedFile(File):
 
-    supportedAlgorithms = {
-        'sha512'
-    }
+    # deprecated, use module-level constant instead
+    supportedAlgorithms = SUPPORTED_ALGORITHMS
```

Comment:
Generally speaking, when attribute are deprecated, I wonder if we should add property with the name `supportedAlgorithms`  and report a deprecation warning ...